README.md

```diff
@@ -102,6 +102,27 @@ sales_2019 |> Quantity.sum(-2, "EUR")
 # {:ok, ~Q[1462.89 EUR]}
 ```
 
+#### Multiplication and division
+
+Quantity has support for single-level complex units that are divided or multiplied:
+
+```elixir
+cost = ~Q[20.00 $]
+amount = ~Q[50 banana]
+cost_per_banana = Quantity.div(cost, amount)
+# ~Q[0.40 $/banana]
+
+new_amount = ~Q[2000 banana]
+Quantity.mult(cost_per_banana, new_amount)
+# ~Q[800.00 $]
```

Comment:
I think this example could be added to the top brief examples in README.md too

Reply:
Agreed force-pushed the readme commit (cf49e104475328619fd31b8ef60a1d6cfa0f29b2) with that change.